### PR TITLE
assimp 3.2

### PR DESCRIPTION
**Pull request by yig**
### All Submissions:

- [yes] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?

### New Formulae Submissions:

- [yes] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [yes] Have you built your formula locally prior to submission with `brew install <formula>`?

### Formulae Updates/Changes:

- [n/a] If your changes include a new `patch` have you submitted the patch upstream & included a brief explanation/link to the upstream submission in the formula?

### Changes to Homebrew's Core:

- [n/a] Have you added an explanation of what your changes do and why you'd like us to include them?
- [n/a] Have you written new tests for your core changes, as applicable? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031) if you'd like one.
- [yes] Have you successfully ran `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Formula/assimp.rb [hunk 1]
```diff
@@ -21,7 +20,9 @@ class Assimp < Formula
   depends_on "boost" => [:recommended, :build]
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    args << "-DASSIMP_BUILD_TESTS=OFF"
```
Comment (bfontaine) [inline Library/Formula/assimp.rb]: Why is this necessary?

Comment (yig) [inline Library/Formula/assimp.rb]: Building the tests downloads gtest, but the gtest repository has moved since this version (3.2) of assimp was released, so building the tests fails.
Building from HEAD doesn't have this problem. When the next assimp release happens, we can turn the tests back on and run the `unit` program that gets generated to test for success. (We wouldn't want to link it, though.)

Comment (bfontaine) [inline Library/Formula/assimp.rb]: Thanks.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @yig; we appreciate it! :tada:
